src/main/drivers/accgyro/accgyro_spi_bmi160.c

```diff
@@ -147,47 +147,47 @@ static int32_t BMI160_Config(const busDevice_t *bus)
 {
 
     // Set normal power mode for gyro and accelerometer
-    spiBusWriteRegister(bus, BMI160_REG_CMD, BMI160_PMU_CMD_PMU_GYR_NORMAL);
+    spiWriteReg(bus, BMI160_REG_CMD, BMI160_PMU_CMD_PMU_GYR_NORMAL);
     delay(100); // can take up to 80ms
 
-    spiBusWriteRegister(bus, BMI160_REG_CMD, BMI160_PMU_CMD_PMU_ACC_NORMAL);
+    spiWriteReg(bus, BMI160_REG_CMD, BMI160_PMU_CMD_PMU_ACC_NORMAL);
```

Comment:
SPI API explanatory comments added in `bus_spi.h`, notably:

```
/*
 * Routine naming convention is:
 * 	spi[Read][Write][Reg][Msk][Buf][RB]
 *
 * 		Read: Perform a read, returning the value read unless 'Buf' is specified
 * 		Write Perform a write
 * 		ReadWrite: Perform both a read and write, returning the value read unless 'Buf' is specified
 * 		Reg: Register number 'reg' is written prior to the read being performed
 * 		Msk: Register number is logically ORed with 0x80 as some devices indicate a read by accessing a register with bit 7 set
 * 		Buf: Pass data of given length by reference
 * 		RB:  Return false immediately if the bus is busy, otherwise complete the access and return true
 */
```